backends/dpdk/dpdkHelpers.h

```diff
@@ -149,6 +149,7 @@ class ConvertStatementToDpdk : public Inspector {
     }
     void branchingInstructionGeneration(cstring true_label, cstring false_label,
                                         const IR::Expression *expr);
+    bool isAssignToDrop(const IR::Expression*);
```

Comment:
Is `isAssignToDrop` used anywhere any more, or is this obsolete?

Reply:
it's obsolete, removed.